index.html

```diff
@@ -54,8 +54,181 @@ <h2>I'm a software developer</h2>
       </ul>
       <a href="#"><img class="scroll-icon" src="image/Disabled.png" alt="Scroll image"></a>
     </section>
-  </main>
-
 
+    <section class="recent-works">
+      <div class="recent-works-title">
+        <h2>My Recent works</h2>
+        <img src="image/Indicator.png" alt="indicator">
+      </div>
+    </section>
+    <section class="my-work-detail">
+      <div class="grid-container">
+        <div class="my-work-card">
+          <div class="card-image"></div>
+          <div class="my-works-text">
```

Comment:
- [ ] As you maybe know a `section` tag groups the generic block of related contents. That's why I would suggest you wrap your `<div class="recent-works-title">` and the `div class="grid-container"` in the same section instead of separation them in differents `section` tags.